### DISCUSSION TITLE
How to install new version of pycaret i.e. pycaret 2.2?

### DISCUSSION THREAD

**Discussion by dryogibansal** [pycaret/pycaret#1881] [Q&A] [answered]
How can I check what version of pycaret is installed? I am trying to use a cross_validation parameter in setup function in version 2.2 (I think I have version 2.2 but not sure. How to check the installed version?).  I am getting this type error.

TypeError: setup() got an unexpected keyword argument 'cross_validation'

 How to resolve this?

**Comment by moezali1** [comment, marked as answer]
@vickaryan Can you please update to the latest version of PyCaret i.e. `2.3.5`. You can check the version of PyCaret like this:

```
from pycaret.utils import version
version()
